pom.xml

```diff
@@ -0,0 +1,21 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" 
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <groupId>org.optaweb.tsp</groupId>
+    <artifactId>optaweb-tsp-parent</artifactId>
```

Comment:
artfictId of any pom should match the directory name (unless there's a good reason not to do that), let's name this optaweb-traveling-salesman.